tests/test_service_v2.py

```diff
@@ -1563,6 +1606,77 @@ def test_batch_request(service):
     assert len(chset_response) == 1
     assert chset_response[0] is None   # response to update request is None
 
+@responses.activate
+def test_batch_request_with_decoded_path(service):
```

Comment:
what does "decoded" means in this context? I understand the PR as that the batch can be "encoded path" or "not encoded path", I do not understand the "decoded" test case at all from its name.
Can you pls explain if I understand something wrong / rename if it is really a batch_request_not_encoded_path scenario?

Reply:
Yes, ..not_encoded is a better naming. I changed it, Thank you!